Source/WebCore/rendering/style/StyleGridData.h

```diff
@@ -168,6 +177,9 @@ class StyleGridData : public RefCounted<StyleGridData> {
     bool m_subgridRows;
     bool m_subgridColumns;
 
+    bool m_masonryRows;
```

Comment:
Might be worth combining this into m_subgridRows with a tri-state enum (normal, subgrid, masonry), so that the invalid state of both masonry+subgrid can't be represented.